docs/tutorial.md

```diff
@@ -0,0 +1,120 @@
+  - **-h, --help**: Show the help message and exit.
+  - **--input=INFILE**: Specifies the urdf file to convert.
+  - **--output=OUTFILE**: Specifies the name of the resulting PROTO file.
+  - **--normal**: If set, the normals are exported if present in the URDF definition.
+  - **--box-collision**: If set, the bounding objects are approximated using boxes.
+  - **--disable-mesh-optimization**: If set, the duplicated vertices are not removed from the meshes (this can speed up a lot the conversion).
+  - **--multi-file**: If set, the mesh files are exported as separated PROTO files.
+  - **--static-base**: If set, the base link will have the option to be static (disable physics)
+  - **--tool-slot=LinkName**: Specify the link that you want to add a tool slot to (exact link name from urdf).
+
+<br /> 
+After your file has been converted, you should have something like this:
+
+![converted files](./images/converted_files.png)
+
+Move these files to to your project’s proto directory (Of course you can do this step by using your OS's GUI)
+
+```
+cp -r KukaLbrIiwa14R820* ~/my_simulation/protos/
+```
+
+<br /> 
+
+## Loading converted model in WEBOTS
+
+Launch Webots and your project world. Click on the plus sign to add your model.
+
+![add node](./images/webots_gui_1.png)
+
+You should see your newly converted model under `PROTO nodes (Current Project)`.
+Select it and click `Add`
+
+![add model](./images/webots_gui_2.png)
+
+It should look something like this:
+
+![add model](./images/webots_robot_sideways.png)
+
+
+As you can see, there are a few things we need to fix. We want the robot to stand upright and perhaps have a fixed base by default. To make these changes, right-click on the robot in the Scene Tree and select `View PROTO Source`.
+This should open the PROTO file in Webot’s text editor and should look something like this:
+
+![add model](./images/kuka_proto.png)
+
+
+The fields in the header correspond to the fields (options) we see in the Scene Tree (compare image above and below)
+
+![add model](./images/kuka_scene_tree.png)
+
+In order to change the default orientation of our model, we have to change the `rotation` field in the proto file. Most urdf models use the z-axis as `up`, while Webots, by default, uses the y-axis as `up`. In this specific case, change the line:
+
+`field  SFRotation  rotation        0 1 0 0`
+to
+`field  SFRotation  rotation     1 0 0 -1.5708`
+
+If these are not the correct values, you can manually adjust the values of your robot node in the scene tree (left side), until the robot is positioned correctly. Then simply copy paste the values into the `PROTO source`. 
+
+Finally, to change the default of the `staticBase` field to `TRUE`, simply change the corresponding line in the `PROTO source` header.
+
+Dont forget to save the file (ctrl + s). In order to see the changes to your proto file in action, either save your world and reload it, or delete the robot and add it again.
```

Comment:
```suggestion
Dont forget to save the file (<kbd>ctrl</kbd> + <kbd>s</kbd>). In order to see the changes to your PROTO file in action, either save your world and reload it, or delete the robot and add it again.
```

Reply:
I cannot commit this change, it is outdated?!